File: HackerRank/Problem_Solving/plusMinus.js
function plusMinus(arr) {
  // Write your code here
  let lt0 = 0;
  let gt0 = 0;
  let zeros = 0
  const len = arr.length;
  arr.forEach((item) => {
      if(item === 0) zeros++;
      else if (item > 0) gt0++;
      else lt0++; 
  });
  
  console.log(gt0/len);
  console.log(lt0/len);
  console.log(zeros/len);

}

arr = [-4, 3, -9, 0, 4, 1];
plusMinus(arr);

File: HackerRank/Problem_Solving/py/day1/plus_minus.py
def plusMinus(arr):
    lt0 = 0
    gt0 = 0
    zeros = 0
    length = len(arr)

    for i in arr:
        if i == 0: zeros += 1
        elif i > 0: gt0 += 1
        else: lt0 += 1

    print(gt0/length)
    print(lt0/length)
    print(zeros/length)

plusMinus([-4, 3, -9, 0, 4, 1])
